feat: add withComment method to add comments (#7083)
doctrine/dbal: docs/en/reference/query-builder.rst

Values of parameters used in a CTE should be defined in the main QueryBuilder.

Comments

> ~~~~~~~~~~~

Reviewer: ```suggestion
~~~~~~~~
```

> To add comments to the query you can use the ``withComment()`` method which will add the comment at the top of the query:

Reviewer: ```suggestion
To add comments to the query, use the ``withComment()`` method which will add the comment before the query:
```
